### PR TITLE
Typescript style improvements

### PR DESCRIPTION
**Pull request by oehm-smith**

### CODE REVIEW

**Comment by lbodor** [comment]
Rebase

File: src/client/service-worker-controller.ts [hunk 1]
```diff
@@ -2,11 +2,11 @@
 // - found Events to use by looking at typings/globals/service_worker_api/index.d.ts and
 //   looking for the 'on' event functions.
 
-var debugEvent = function (event:any) {
+function debugEvent (event:any): void {
```
Comment (lbodor) [inline src/client/service-worker-controller.ts]: What is the type of event?

File: src/client/service-worker-controller.ts [hunk 2]
```diff
@@ -45,16 +45,16 @@ var cleanCache1 = function (event:any) {
   );
 };
 
-self.addEventListener('install', function (event:InstallEvent) {
+self.addEventListener('install', (event:InstallEvent) => {
   debugEvent(event);
 });
```
Comment (lbodor) [inline src/client/service-worker-controller.ts]: Could these be written as self.addEventListener('install', debugEvent)?

**Comment by oehm-smith** [comment]
It appears to work.  Code updated.

**Pull request merged**